dbms/src/Flash/Planner/PhysicalPlan.cpp

```diff
@@ -93,6 +93,11 @@ void PhysicalPlan::build(const tipb::Executor * executor)
 {
     RUNTIME_CHECK(executor);
     RUNTIME_CHECK(executor->has_executor_id());
+    // For plans generated by TiDB, assume 1st hashagg can only be followed by ExchangeSender.
```

Comment:
If TiDB is using cop/batchcop request, will it enable auto pass optimazation?

Reply:
TiDB will only enable auto pass through hash agg for MPP. Maybe we can add a check